src/main/surfrdMod.F90

```diff
@@ -300,6 +301,48 @@ subroutine surfrd_get_num_patches (lfsurdat, actual_maxsoil_patches, actual_numc
 
   end subroutine surfrd_get_num_patches
 
+!-----------------------------------------------------------------------
+  subroutine surfrd_get_nlevurb (lfsurdat, actual_nlevurb)
```

Comment:
> Created 0.9x1.25 surface datasets and landuse timeseries for 1850, 1850-2015, 2000, 2005, 2015-2100 (SSP1-1.9, SSP1-2.6, SSP2-4.5, SSP3-7.0, SSP4-3.4, SSP4-6.0, SSP5-8.5) using mksurfdata_esmf.

You performed thorough testing using mksurfdata_esmf. Did you subsequently start a model simulation to confirm that your mods worked? I'm guessing that system testing would not work unless you also pointed to the newly created fsurdat files.

Reply:
Yes, I successfully ran a historical simulation (at least for 20 years, the simulation is ongoing) and pointed to the new surface dataset and landuse timeseries file in user_nl_clm.  I didn't want to spend a lot of computer time running simulations for all of those scenarios.  I'm also not that familiar with the tools system testing.

Reply:
Sounds good to me.

For system testing, I was referring to the ./run_sys_tests rather than the tools testing. Again, though, I wouldn't worry about it because many (most? all?) of the tests would require that you point them to your new fsrudat files.

Reply:
Sam should generate a temporary set of fsurdat files to run ./run_sys_tests

Reply:
> Sam should generate a temporary set of fsurdat files to run ./run_sys_tests

IN PROGRESS: Generating all fsurdat files using `mksurfdata_jobscript_multi`

Reply:
Started cheyenne's test-suite pointing to the default fsurdat files (not, yet, the new ones that I'm generating). I realized that the old files already include nlevurb. Keith had made that clear in note (1c) above (sorry, I forgot having read that). 

Tests are passing, though some differ from the baseline (dev090) because we're pointing to this new file as explained in note 1b above:  `CTSM52_tbuildmax_OlesonFeddema_2020_0.9x1.25_simyr1849-2106_c200605.nc`

So now I'm rethinking the purpose of this test. I will rerun the test-suite without the new file from note 1b to confirm no diffs from baseline.

Then there's the new urban raw data (note 1a) going into the fsurdat files that I am generating. I vote against running the full test-suite for this. We have several new raw datasets coming with #1732 . I expect each one will lead to diffs from the baseline. I have not started generating new fsurdat files for the other new raw datasets. I prefer to do that once in the end for all of them. The complete generation of new surface datasets is time consuming.

I'm open to differing views.

Reply:
Discussed this with @ekluzek and agreed that the above approach is fine:
- Wait to generate new datasets once at the end.
- No need to rerun the test-suite with the old tbuildmax file.

Reply:
The cheyenne test-suite finished and two tests FAIL:
`ERP_Ld9.f45_g37.I2000Clm51Bgc.cheyenne_intel.clm-default`
`ERP_Ld9.f45_f45_mg37.I2000Clm50FatesRs.cheyenne_intel.clm-FatesColdDefAllVars`
due to temporary mods we made in Externals.cfg in the ctsm5.2.mksurfdata branch, so nothing to do with the GaoOneill branch of this PR.